client/state/plugins/wporg/reducer.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import { RECEIVE_WPORG_PLUGIN_DATA, FETCH_WPORG_PLUGIN_DATA } from 'state/action-types';
+
```

Comment:
well, it's an empty object / dictionary... we could explicitly declare it, but I thought it was not necessary yet